georaster-layer-for-leaflet.js

```diff
@@ -442,6 +446,56 @@ const GeoRasterLayer = L.GridLayer.extend({
     }
   },
 
+  /**
+   * The callback used to determine the colour based on the values of each pixel
+   *
+   * @callback pixelValuesToColor
+   * @param {array | number} values - The pixel value
+   * @returns {string} - Any valid CSS color string
+   */
+
+  /**
+   * Redraws the active map tiles updating the pixel values using the supplied callback
+   * @param {pixelValuesToColor} pixelValuesToColor - Callback that handles getting the pixel color
+   */
+  updateColors(pixelValuesToColor) {
```

Comment:
Could we change to `updateColors({ pixelValuesToColorFn })`?  I want to add flexibility to add additional parameters easily and not worry about the order of parameters.  For example, (definitely doesn't have to be now), but I might like to add a custom `debugLevel` parameter in the future that can override the class' debugLevel for just this function.

Reply:
In this scenario, would it not be better to leave the first param as the updated colour function given its the primary usecase for this method. Then have a second options parameter for the local `debugLevel` etc?

It could look like `updateColors(pixelValuesToColorFn, { debugLevel = 0, ...rest })`